changelog/unreleased/bugfix-pagination-after-increasing-items-per-page

```diff
@@ -0,0 +1,6 @@
+Bugfix: Pagination after increasing items per page
+
+An issue where the file list incorrectly showed no items after paginating and increasing the amount of items per page has been fixed.
```

Comment:
```suggestion
An issue where the file list incorrectly showed no items after paginating and changing the amount of items per page has been fixed.
```

Reply:
Was about to do the same, but the bug report is still right though, no? The issue only occurred when increasing the amount of items.

Reply:
yes :D fine by me either way